yb-voyager/src/dbzm/schemaRegistry.go

```diff
@@ -115,3 +122,21 @@ func (sreg *SchemaRegistry) Init() error {
 	}
 	return nil
 }
+
+func (sreg *SchemaRegistry) getAndStoreTableSchema(tableName string) (*TableSchema, error) {
+	schemaFilePath := filepath.Join(sreg.exportDir, "data", "schemas", sreg.exportSourceType, fmt.Sprintf("%s_schema.json", tableName))
+	schemaFile, err := os.Open(schemaFilePath)
+	defer func() {
```

Comment:
Nit: `defer schemaFile.Close()`

Reply:
Ah, it wouldn't let me do this because the err is not captured. That's why I had to create an in-line func